packages/neon-core/__integration__/rpc/clients/NeoServerRpcClient.ts

```diff
@@ -18,18 +21,18 @@ beforeAll(async () => {
   const url = await TestHelpers.getIntegrationEnvUrl();
   client = new rpc.NeoServerRpcClient(url);
 
-  let height = 0;
-  while (!txid) {
-    const block = await client.getBlock(height, true);
-    if (block.tx.length !== 0 && block.tx[0].hash) {
-      txid = block.tx[0].hash;
-      blockhash = block.hash;
-      blockHeight = height;
-      return;
-    }
-
-    height++;
-  }
+  // let height = 0;
+  // while (!txid) {
+  //   const block = await client.getBlock(height, true);
+  //   if (block.tx.length !== 0 && block.tx[0].hash) {
+  //     txid = block.tx[0].hash;
+  //     blockhash = block.hash;
+  //     blockHeight = height;
+  //     return;
+  //   }
+
+  //   height++;
+  // }
```

Comment:
@raulduartep What is this commented code?

Reply:
i forgot to uncomment, it caused problems to run local